src/main/resources/assets/spectrum/lang/zh_cn.json

```diff
@@ -1,1818 +1,3730 @@
+    "item.spectrum.ring_of_pursuit": "探求之戒",
+    "item.spectrum.ring_of_pursuit.tooltip": "增加你的挖掘速度",
+
+    "item.spectrum.crafting_tablet": "合成模板",
+    "item.spectrum.crafting_tablet.tooltip.no_recipe": "将物品添加到合成板中以设置配方",
+    "item.spectrum.crafting_tablet.tooltip.crafting_recipe": "按下使用键合成",
+    "item.spectrum.crafting_tablet.tooltip.shift_to_view_gui": "潜行并按下使用键来打开设置界面",
+    "item.spectrum.crafting_tablet.tooltip.pedestal_recipe": "于七彩基座中使用以自动合成",
+    "item.spectrum.crafting_tablet.tooltip.recipe": "%d %s",
+
+    "item.spectrum.star_fragment": "流星碎片",
+    "item.spectrum.stardust": "星尘",
+    "item.spectrum.frigid_stardust": "冰冷星尘",
+    "item.spectrum.frigid_stardust.tooltip": "§o§9温和的力量",
+    "item.spectrum.hibernating_jade_vine_bulb": "沉眠的玉藤种子",
+    "item.spectrum.hibernating_jade_vine_bulb.tooltip": "有些……死气沉沉？",
+    "item.spectrum.germinated_jade_vine_bulb": "焕活的玉藤种子",
+    "item.spectrum.germinated_jade_vine_bulb.tooltip": "§7依附在木栅栏上生长",
+    "item.spectrum.germinated_jade_vine_bulb.tooltip2": "§7只在§f月光§7下生长",
+    "item.spectrum.germinated_jade_vine_bulb.tooltip3": "§e⚠ 暴露在阳光下会使其枯萎",
+    "item.spectrum.jade_vine_petals": "玉藤花瓣",
+    "item.spectrum.jadeite_petals": "翡翠花瓣",
+  
+    "item.spectrum.moonstruck_nectar": "望舒琼浆",
+    "item.spectrum.moonstruck_nectar.tooltip": "它能修复任何损坏的工具",
+    "item.spectrum.moonstruck_nectar.tooltip2": "饮用后获得短时间的生命恢复爆发",
+    "item.spectrum.restoration_tea": "恢复之茶",
+    "item.spectrum.restoration_tea.tooltip": "§7祛除所有负面效果",
+    "item.spectrum.restoration_tea.tooltip2": "§7并使你暂时免疫负面效果",
+    "item.spectrum.restoration_tea.tooltip_milk": "加奶",
+    "item.spectrum.jade_jelly": "玉藤果冻",
+    "item.spectrum.jade_jelly.tooltip": "营养与风味俱佳",
+    "item.spectrum.glistering_jelly_tea": "闪烁的玉藤果冻茶",
+    "item.spectrum.glistering_jelly_tea.tooltip": "暂时增加你的生命值",
+    "item.spectrum.enchanted_golden_carrot": "附魔金胡萝卜",
+    "item.spectrum.jaramel": "玉藤焦糖",
+    "item.spectrum.lucky_roll": "幸运奶心卷",
+    "item.spectrum.triple_meat_pot_pie": "肉夹馍",
+    "item.spectrum.star_candy": "星糖",
+    "item.spectrum.star_candy.tooltip": "祛除单一的负面状态效果",
+    "item.spectrum.purple_star_candy": "星糖",
+    "item.spectrum.purple_star_candy.tooltip": "祛除所有负面效果",
+    "item.spectrum.purple_star_candy.tooltip2": "并恢复全部生命值和饱食度",
+    "item.spectrum.suspicious_brew": "可疑的酒",
+    "item.spectrum.suspicious_brew.tooltip": "里面可能混杂了什么？",
+    "item.spectrum.suspicious_brew.tooltip.preview": "酒精度越高，持续时间越长 %",
+    "item.spectrum.suspicious_brew.tooltip.preview2": "效果取决于使用的花朵",
+    "item.spectrum.jade_wine": "玉酒",
+    "item.spectrum.jade_wine.tooltip.bloominess": "%d %% 花开度",
+    "item.spectrum.jade_wine.tooltip.sweetened": "加糖",
+    "item.spectrum.jade_wine.tooltip.preview3": "酒精含量越高，正面效果越多、时间越长。 %",
+    "item.spectrum.jade_wine.tooltip.preview": "用玉藤种子和花瓣混合酿造",
+    "item.spectrum.jade_wine.tooltip.preview2": "花瓣比玉藤种子发酵慢，但减少了负面影响",
+    "item.spectrum.pure_alcohol": "高纯度酒精",
+    
```

Comment:
This is missing. The joke here is a reference to Tabletop RPG's like Dungeons and Dragons, where your Intelligence Modifier is about the regular limit (usually +5)

Reply:
Thanks for the correction. I have another question for you. Is there anything special about the glass_crest series equipment? Is it an upgraded version of the malachite equipment? I don't know how to translate it correctly.

Reply:
> Is it an upgraded version of the malachite equipment
yes

Reply:
> > Is it an upgraded version of the malachite equipment
> 
> yes

Why is it called glass_crest? Does it have any special meaning?